test/functional/test_site/index.md

```diff
@@ -219,3 +219,9 @@ tags: ["tag-frontmatter-shown", "tag-included-file", "+tag-exp*", "-tag-exp-hidd
 <div id="test-markbind-plugin">
   Markbind Plugin Pre-render Placeholder
 </div>
+
+# Test search indexing
```

Comment:
Hmm, "Level 1 header without attributes specified should be indexed" to be more consistent with the rest?

Reply:
I intended for this to be a "heading" which describes the section containing the two headings below it. This is mainly because the sidebar links have indentation based on the heading level, and without this level 1 heading, the links generated below will appear to be under the "Markbind Plugin Pre-render‎" section instead.

<img width="245" alt="Screenshot 2019-03-14 at 4 40 24 PM" src="https://user-images.githubusercontent.com/17447681/54342664-06771c80-4678-11e9-81fb-f2d82c344153.png">

With this in mind, I'm not exactly sure if "Level 1 header without attributes specified should be indexed" would accurately describe this section 😅

Reply:
I see, my bad then, I thought this was part of the test cases at first glance.